### PR TITLE
Map the Facility Locator over to /find-locations/ in VA.gov

### PR DESCRIPTION
**Pull request by ncksllvn**
## Description
This pull requests updates the Facility Locator to use a derived `rootUrl` via a `manifest.js` so that the application can live at `/find-locations/` on Brand Consolidation builds. It also updates all VA.gov-only pages/data to use the new URL, and uses the `manifest.js` for application links. This goes hand-in-hand with https://github.com/department-of-veterans-affairs/vagov-content/pull/93, which needs to be merged first because the link-checker will continue to fail this PR until then.

Moves to validate - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14672

## Testing done
Tested the application locally 

## Screenshots
image

## Acceptance criteria
- [x] The new URL works and does not negatively impact the Facility Locator

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/auth/containers/AuthApp.jsx [hunk 1]
```diff
@@ -138,7 +140,11 @@ export class AuthApp extends React.Component {
                 you with the right person who can help.
               </p>
               <p>
-                <a href="/facilities/?facilityType=health&page=1&zoomLevel=7">
+                <a
+                  href={`${
+                    facilityLocator.rootUrl
+                  }/?facilityType=health&page=1&zoomLevel=7`}
+                >
```
Comment (ncksllvn) [inline src/applications/auth/containers/AuthApp.jsx]: I don't always agree with Prettier 😕

**Comment by ncksllvn** [comment]
@kfrz Are there updates that need to be made here in the FE metrics [whitelist](https://github.com/department-of-veterans-affairs/vets-website/blob/f63c663a1cab4aca373db0f43563d5b846575918/src/platform/monitoring/frontend-metrics/whitelisted-paths.js)?

**Comment by jbalboni** [comment]
@ncksllvn I think there's one missed instance in sitemap-helpers.js

**Comment by ncksllvn** [comment]
@jbalboni I'm not completely sure, but I think that file is executed only while running under Vets.gov.

**Comment by jbalboni** [comment]
@ncksllvn It might be for now, but you may as well add this path to the list so it doesn't break later.

**Comment by ncksllvn** [comment]
@jbalboni Ah I see what you mean. Added it ✅

**Review by jbalboni — APPROVED** [review]
Looks good, except for the failing build

**Comment by ncksllvn** [comment]
@jbalboni 😆 I just merged the content PR and reran the build here. Hopefully that resolves it 🤞

**Pull request merged**